Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -40,85 +42,86 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
-  const auto InputImage_nosform_direction = InputImage_nosform->GetDirection();
+  const auto inputImageDirection = inputImage->GetDirection();
+  const auto inputImageNoQformDirection = inputImageNoQform->GetDirection();
+  const auto inputImageNoSformDirection = inputImageNoSform->GetDirection();
 
   // Verify that the sform from InputImage being used when reading an image with both sform and qform
   // The sforms should be identical in both cases.
-  if (InputImage_direction != InputImage_noqform_direction)
+  if (inputImageDirection != inputImageNoQformDirection)
   {
     std::cerr << "Error: direction matrix from the image with sform and qform should be identical to ";
     std::cerr << "the matrix from the image with sform only" << std::endl;
-    std::cerr << "sform and qform image direction:" << std::endl << InputImage_direction << std::endl;
-    std::cerr << "sform-only image direction:" << std::endl << InputImage_noqform_direction << std::endl;
+    std::cerr << "sform and qform image direction:" << std::endl << inputImageDirection << std::endl;
+    std::cerr << "sform-only image direction:" << std::endl << inputImageNoQformDirection << std::endl;
     return EXIT_FAILURE;
   }
   // Verify that the qform alone is not identical to the sform (due to lossy storage capacity)
   // This difference is small, but periodically causes numerical precision errors,
   // VERIFY THAT THE DIRECTIONS ARE ACTUALLY DIFFERENT, They are expected to be different due to lossy
   // representation of the qform.
-  if (InputImage_direction == InputImage_nosform_direction)
+  if (inputImageDirection == inputImageNoSformDirection)
   {
     std::cerr << "Error: direction matrices from sform and qform should differ because of lossy ";
     std::cerr << "storage methods." << std::endl;
-    std::cerr << "sform and qform image direction:" << std::endl << InputImage_direction << std::endl;
-    std::cerr << "qform-only image direction:" << std::endl << InputImage_nosform_direction << std::endl;
+    std::cerr << "sform and qform image direction:" << std::endl << inputImageDirection << std::endl;
+    std::cerr << "qform-only image direction:" << std::endl << inputImageNoSformDirection << std::endl;
     return EXIT_FAILURE;
   }
 
   // Write image that originally had no sform direction representation into a file with both sform and qform
-  const std::string                            test_output_dir = av[4];
-  const std::string                            test_filename = test_output_dir + "/test_filled_sform.nii.gz";
+  const std::string                            testOutputDir = av[4];
+  const std::string                            testFilename = testOutputDir + "/test_filled_sform.nii.gz";
   itk::ImageFileWriter<TestImageType>::Pointer writer = itk::ImageFileWriter<TestImageType>::New();
-  writer->SetFileName(test_filename);
-  writer->SetInput(InputImage_nosform);
-  writer->Update();
+  writer->SetFileName(testFilename);
```

Comment:
why not `ITK_TRY_EXPECT_NO_EXCEPTION(itk::WriteImage(inputImageNoSform, testFilename));`?

Reply:
Done, thanks

Reply:
:+1: Done in ceac451.